clients/cobol-lsp-vscode-extension/src/services/ProcessorGroups.ts

```diff
@@ -286,3 +326,26 @@ async function loadProcessorGroupSettings<T extends string | string[]>(
     return configObject;
   }
 }
+
+export async function prepareProcessorGroupConfigPathsForEndevor(
```

Comment:
I think this function should be just remove/inlined, it does two unrelated things and it's called from exactly one call site.